x-pack/plugin/esql/src/main/generated/org/elasticsearch/xpack/esql/expression/function/scalar/math/HypotEvaluator.java

```diff
@@ -13,29 +13,31 @@
 import org.elasticsearch.compute.data.Page;
 import org.elasticsearch.compute.operator.DriverContext;
 import org.elasticsearch.compute.operator.EvalOperator;
+import org.elasticsearch.compute.operator.Warnings;
 import org.elasticsearch.core.Releasables;
 import org.elasticsearch.xpack.esql.core.tree.Source;
-import org.elasticsearch.xpack.esql.expression.function.Warnings;
 
 /**
  * {@link EvalOperator.ExpressionEvaluator} implementation for {@link Hypot}.
  * This class is generated. Do not edit it.
  */
 public final class HypotEvaluator implements EvalOperator.ExpressionEvaluator {
-  private final Warnings warnings;
+  private final Source source;
```

Comment:
These changes seem unrelated?

Reply:
It looks like these are the changes from https://github.com/elastic/elasticsearch/commit/30ff4741c674532c832c53a3bd31aa307dce8a95 (Commit `30ff474`), I think? I’m not sure why they’re showing up here, but presumably it’s because I did something wrong when merging upstream/main into my branch. Hopefully this most merge removes them from the listed changes.